README.md

```diff
@@ -42,6 +42,7 @@ Download Moveit2_tutorials Source Code
 
     wget https://raw.githubusercontent.com/ros-planning/moveit2_tutorials/main/moveit2_tutorials.repos
     vcs import < moveit2_tutorials.repos
+    git clone https://github.com/ros-planning/moveit2_tutorials.git
     rosdep install -r --from-paths . --ignore-src --rosdistro foxy -y
```

Comment:
Sorry I didnt say but we should change this to clone `moveit2_tutorials` and then extract the `.repos` from that folder and not use `wget`.

```
git clone https://github.com/ros-planning/moveit2_tutorials.git
vcs import < moveit2_tutorials/moveit2_tutorials.repos
rosdep install -r --from-paths . --ignore-src --rosdistro foxy -y
```

Reply:
Sure